### PR TITLE
Fix python 3.10 compatibility issues

### LINKED ISSUE #195: Incompatibility with Python 3.10
At least the following:

* [ ] TypeError: As of 3.10, the *loop* parameter was removed from Queue() since it is no longer necessary
* [ ] `asyncio.sleep`'s *loop* parameter also got deleted
* [ ] `asyncio.wait`'s *loop* parameter got deleted

### PR DESCRIPTION
**Pull request by thomwiggers**
Closes #195

### CODE REVIEW

**Comment by thomwiggers** [comment]
All these shims aren't getting prettier, so it might be best to just start to pick a cut-off

File: irc3/compat.py [hunk 1]
```diff
@@ -13,9 +14,36 @@
...
     import asyncio
-    from asyncio.queues import Queue
+    from asyncio.queues import Queue as BaseQueue
     from asyncio.queues import QueueFull  # NOQA
+    from asyncio import sleep as _original_sleep
+    from asyncio import wait as _original_wait
 except ImportError:  # pragma: no cover
     import trollius as asyncio  # NOQA
-    from trollius.queues import Queue  # NOQA
+    from trollius.queues import Queue  as BaseQueue # NOQA
```
Comment (gawel) [inline irc3/compat.py:23]: pretty sure trollius can be removed. this was required for py2, maybe py3.3 max (?)

python3.5 should not require this

Comment (thomwiggers) [reply irc3/compat.py:23]: Perhaps that cleanup best happens in a separate PR?

Comment (gawel) [reply irc3/compat.py:23]: I don't really care :) I can merge as this if it's good enough for you

Comment (thomwiggers) [reply irc3/compat.py:23]: Sounds good to me

**Pull request merged**